tests/unit/job.spec.ts

```diff
@@ -2,23 +2,15 @@ import { expect } from 'chai';
 import { describe, it } from 'mocha';
 import { MongoClient, ObjectId } from 'mongodb';
 import { DropAllCollections } from '../../src/router.debug';
-import { ChaiRequest } from '../config';
+import { ChaiRequest, TEST_DB_CLIENT } from '../config';
 
 require('../config'); // recommended way of loading root hooks
 
 const URL = '/jobs';
 
-const DB_URL: string = process.env.MONGO_DB_URI!;
-
-const CLIENT = new MongoClient(DB_URL);
-
 describe('# JOB', () => {
-  before(async function () {
-    await CLIENT.connect().catch(console.error);
-  });
-  afterEach(async function () {
-    await DropAllCollections();
-  });
+  beforeEach(async () => await DropAllCollections());
```

Comment:
Can beforeEach be defined higher up? Otherwise each test suite will have to repeat this.